lib/code_corps/messages/conversation_query.ex

```diff
@@ -41,8 +41,8 @@ defmodule CodeCorps.Messages.ConversationQuery do
   - belong to a `CodeCorps.Message` initiated by admin, with at least a single
     reply in the form of a `CodeCorps.ConversationPart`
   """
-  @spec status_filter(Queryable.t, map) :: Queryable.t
-  def status_filter(queryable, %{"status" => "active"}) do
+  @spec active_filter(Queryable.t, map) :: Queryable.t
+  def active_filter(queryable, %{"active" => true}) do
```

Comment:
@joshsmith is this how active filter should work?  What specifically filters out inactive records?